src/computed-style-property-map.js

```diff
@@ -78,15 +103,28 @@
 
     switch (property) {
```

Comment:
This switch is getting large. Do we need it? ComputedStylePropertyMap.get throws for unsupported properties. Can we just do
```
var value = this.get(property);
if (Array.isArray(value)) {
    // TODO: Handle shorthands
    throw new TypeError('Not implemented yet');
} else {
    return value ? [value] : [];
}
```
here?